src/components/Dashboard/Dashboard.jsx

```diff
@@ -49,43 +52,92 @@ export class Dashboard extends React.Component {
                             </div>
                         </header>
                         <main className={css.main}>
-                            <div className={css.button_and_table}>
-                                <div>
-                                    <table className='table'>
-                                        <thead>
-                                        <tr className="table-active">
-                                            <th scope="col"></th>
-                                            <th scope="col">Title</th>
-                                            <th scope="col">Owner</th>
-                                            <th scope="col">Key</th>
-                                            <th scope="col">Category</th>
+                            <div className={css.buttons_and_table}>
+                                <table className={css.table}>
+                                    <thead>
+                                    <tr>
+                                        <th></th>
+                                        <th>Title</th>
+                                        <th>Owner</th>
+                                        <th>Key</th>
+                                        <th>Category</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    {this.state.items.slice(this.state.page * this.state.amount, (this.state.page + 1) * this.state.amount).map(item => {
```

Comment:
Difficult to read.
Move `this.state.items.slice(this.state.page * this.state.amount, (this.state.page + 1) * this.state.amount)`
Into separated variable:
```
render() {
        const {items, page, amount} = this.state;
        const pageBoards = items.slice(page * amount, (page + 1) * amount);

...
```